data/json/item_groups.json

```diff
@@ -2759,12 +2760,15 @@
       [ "recipe_elfa", 1 ],
       [ "jug_plastic", 10 ],
       [ "spray_can", 50 ],
+      [ "keg", 10 ],
       [ "jar_glass", 10 ],
       [ "jar_3l_glass", 8 ],
       [ "flyer", 10 ],
       [ "scorecard", 10 ],
       [ "talking_doll", 60 ],
       [ "marble", 65 ],
+      [ "55gal_drum", 1 ],
+      [ "30gal_drum", 2 ],
```

Comment:
I though I removed these for good in #28977?

Reply:
> 
> 
> I though I removed these for good in #28977?

That was an error during conflict resolution. It will be fixed in #28958.